lib/msal-node/docs/certificate-credentials.md

```diff
@@ -35,14 +35,18 @@ For more information, see: [Register your certificate with Microsoft identity pl
 
 ```javascript
 const msal = require('@azure/msal-node');
+// Due to security reasons, secrets should not be hardcoded.
```

Comment:
Can we instead add this as a tip above:

> 💡 ⚠️ Due to security reasons, secrets should not be hardcoded. The dotenv npm package can be used to store secrets in a .env file (located in project's root directory) and should be included in .gitignore to prevent accidental uploads of the secrets.